README.md

```
@@ -47,18 +53,41 @@ O Busca Pet é uma aplicação web desenvolvida para ajudar na localização de
    ```
 
+4. Inicie o servidor de desenvolvimento:
+
+   ```
+   pnpm run start:dev
+   ```
+
+
 ### Estrutura de arquivos e pastas
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)